README.md

```diff
@@ -158,6 +158,7 @@ Table of Contents
 ## Automated Browser Testing
 
   * [gridlastic.com](https://www.gridlastic.com/) — Selenium Grid testing with free plan up to 4 simultaneous selenium nodes/10 grid starts/4,000 test minutes/month
+  * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
```

Comment:
@ripienaar I believe it'll be better to move it from CI / CD to Automated Browser Testing - as it is primary purpose is much more about testing, not Continuous Integration or Continuous Delivery.

Will it be ok if I'll delete it from CI / CD section?